src/utils/getNewRect.spec.js

```diff
@@ -0,0 +1,86 @@
+import getNewRect from './getNewRect';
+import Action from './Action';
+
+describe('getNewRect', () => {
+  const rect = { x: 100, y: 150, width: 200, height: 300 };
+
+  test('should create identical rect when using zero offset and full size', () => {
+    const result = { x: 100, y: 150, width: 200, height: 300 };
```

Comment:
Let's rename this to something along the lines of `expected` or `expectedResult`